w5e3.py

```diff
@@ -0,0 +1,31 @@
+"w5e3"
+#Parseltongue
+"""
+The course logo appears in the resources / logo.jpg file, and inside it are hidden some secret messages.
+The messages are strings at least 5 letters long, written in small English letters only and ending with an exclamation mark.
+Open the logo for reading in a binary configuration, and extract the secret messages from it.
+Keep in mind that the file may be very large, and it is best not to read it all in one go.
+Find help on the Internet regarding opening files in a binary form and a graded reading of the file.
+Be careful not to use techniques we have not learned (or add them only in addition to such a solution).
+"""
+# open file binary mood
+f = open('logo.jpg', 'rb')
+content = f.read()
+str =""
+for i in range(len(content)):
```

Comment:
i and j are bad names, give names that mean something, it makes your code unreadable

Reply:
You are right but I was told that writing i as an index is something relatively understandable that everyone uses it and because j is similar to i it is customary to use them

Reply:
In my opinion, if you have a better alterative, you should avoid using them. in here I might have used the name letter_index, for readability. Also, you might want to use enumerate here